bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -1026,10 +1026,11 @@ public String toString() {
     }
 
     @Override
-    public void add(Item element) {
+    public Item add(Item element) {
         if (itemRegistry != null) {
-            itemRegistry.add(element);
+            return itemRegistry.add(element);
         }
+        return element;
```

Comment:
```text
      * @throws IllegalStateException
       *             if no ManagedProvider is available
```
Should we return element or throw the IllegalStateException as documented in the API?